nes_bnf.html

```diff
@@ -0,0 +1,1206 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

```

Comment:
Można dać
http://htmlpreview.github.io/?https://github.com/stas53/ezd-analizy-it/blob/9bc5087944a5437bf56f175cb5ebe71d0cc2d6ac/nes_bnf.html
ale to trzeba będzie jeszcze raz zmienić, gdy pull request zostanie zaakceptowany, samplik HTML uzyska nową lokalizację.